climpred/prediction.py

```diff
@@ -240,8 +240,12 @@ def compute_hindcast(
 
         if a.time.size > 0:
             logging.info(
-                f'at lead={str(i).zfill(2)}: dim={dim}: {a.time.min().values}-'
-                f'{a.time.max().values}'
+                f'lead={str(i).zfill(2)} | '
+                f'dim={dim} | '
+                f'inits={forecast["time"].min().values}'
+                f'-{forecast["time"].max().values} | '
+                f'verif={a["time"].min().values}'
+                f'-{b["time"].max().values}'
```

Comment:
isnt this breaking if the times here are not the same?

Reply:
I don't understand what you mean here. This is just printing to the log.

So `forecast.time.min/max()` is the originally sliced forecast so it prints out the initializations. `a.min()/max()` is the actual dates.

Reply:
Ah I didn't see the typo between a and b. Maybe that's what you meant.